### PR TITLE
Add planID to the KEB operations metrics

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Added planID to the KEB operations metrics

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/metrics/operations_collector.go [hunk 1]
```diff
@@ -15,68 +17,114 @@ const (
...
+// AZURE LITE:
+
+// - compass_keb_operations_azure_lite_provisioning_failed_total
+// - compass_keb_operations_azure_lite_provisioning_in_progress_total
+// - compass_keb_operations_azure_lite_provisioning_succeeded_total
+// - compass_keb_operations_azure_lite_deprovisioning_failed_total
+// - compass_keb_operations_azure_lite_deprovisioning_in_progress_total
+// - compass_keb_operations_azure_lite_deprovisioning_succeeded_total
 type OperationsStatsGetter interface {
 	GetOperationStats() (internal.OperationStats, error)
```
Comment (polskikiel) [inline components/kyma-environment-broker/internal/metrics/operations_collector.go:49]: just a thought, we can make a struct OperationStatsByPlan which would be `map[string][string]struct{}` type of data.

File: components/kyma-environment-broker/internal/metrics/operations_collector.go [hunk 2]
```diff
@@ -3,80 +3,104 @@ package metrics
 import (
 	"fmt"
 
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/broker"
+
```
Comment (ksputo) [inline components/kyma-environment-broker/internal/metrics/operations_collector.go:7]: unnecessary newline

File: components/kyma-environment-broker/internal/storage/storage_test.go [hunk 3]
```diff
@@ -8,15 +10,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/postsql"
```
Comment (ksputo) [inline components/kyma-environment-broker/internal/storage/storage_test.go:13]: please sort imports here

**Review by ksputo — DISMISSED** [review]

**Review by ksputo — APPROVED** [review]

**Pull request merged**